src/data/services/EnterpriseCatalogApiService.js

```diff
@@ -31,6 +31,10 @@ class EnterpriseCatalogApiService {
     ).get(url);
   }
 
+  static fetchEnterpriseCatalog(catalogUuid) {
+    return EnterpriseCatalogApiService.apiClient().get(`${EnterpriseCatalogApiService.enterpriseCustomerCatalogsUrl}${catalogUuid}`);
```

Comment:
[clarification] Is this function intended to be used by anything? Doesn't appear to be used right now.